.cbmc-batch/jobs/aws_hash_c_string/Makefile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########
+#32: 4s
+#64: 5s
+#128 6s
+#256 11s
+#1024: 2m30s
+MAX_STRING_SIZE ?= 32
+DEFINES += -DMAX_STRING_SIZE=$(MAX_STRING_SIZE) 
+
+UNWINDSET += strlen.0:$(shell echo $$(( $(MAX_STRING_SIZE) + 1)))
+UNWINDSET += hashlittle2.2:$(shell echo $$(( $$(( $(MAX_STRING_SIZE) / 12 )) +1 )) )
+UNWINDSET += __CPROVER_file_local_lookup3_c_hashlittle2.0:$(shell echo $$(( $(MAX_STRING_SIZE) - 11)))
```

Comment:
Two things seem a little bit confusing:
1. You probably don't need both unwind hashlittle2 and unwind `__CPROVER_file_local_lookup3_c_hashlittle2`.
2. The unwind number should probably be `MAX_STRING_SIZE / 12 + 1` for all the hashlittle loops

Reply:
Done.